src/core/annotation.js

```diff
@@ -715,6 +715,8 @@ var TextWidgetAnnotation = (function TextWidgetAnnotationClosure() {
     // Process field flags for the display layer.
     this.data.readOnly = this.hasFieldFlag(AnnotationFieldFlag.READONLY);
     this.data.multiLine = this.hasFieldFlag(AnnotationFieldFlag.MULTILINE);
+    this.data.comb = this.hasFieldFlag(AnnotationFieldFlag.COMB) &&
+                     this.data.maxLen !== null && !this.data.multiLine;
```

Comment:
I think that you're missing a number of field checks here, please see the specification text:
> (PDF 1.5) May be set only if the MaxLen entry is present in the text field dictionary (see Table 229) and ***if the Multiline, Password, and FileSelect flags are clear***. If set, the field shall be automatically divided into as many equally spaced positions, or combs, as the value of MaxLen, and the text is laid out into those combs.